generatebundlefile/hack/release_staging.sh

```diff
@@ -21,8 +21,17 @@ set -o pipefail
 export LANG=C.UTF-8
 
 BASE_AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+BASE_DIRECTORY=$(git rev-parse --show-toplevel)
+
+. "${BASE_DIRECTORY}/generatebundlefile/hack/common.sh"
+ORAS_BIN=${BASE_DIRECTORY}/bin/oras
 
-cat << EOF > configfile
+make build
+chmod +x ${BASE_DIRECTORY}/generatebundlefile/bin/generatebundlefile
+
+regional_build_mode=${REGIONAL_BUILD_MODE:-}
```

Comment:
Only in the non-regional mode (old flow), we push packages to the old packages account, otherwise we skip it and just push to the staging account (`public.ecr.aws/w9m0f3l5`)